react/src/components/carousel/CarouselBottomSection.module.css

```diff
@@ -0,0 +1,247 @@
+        background: transparent url('@/icons/smartphone/Polygon_1_copy.png') 0% top no-repeat padding-box;
+        background-size: contain;
+    }
+
+    #bottomPhotoOverlay2 span {
+        /* Self Layout Properties */
+        top: 2%;
+        left: 5%;
+    }
+}
+
+/* Center bottomPhoto boxes if only one is displayed per row */
+@media only screen and (min-width: 600px) and (max-width: 746px) {
+    #bottomPhotoContainer {
+        /* Content Layout Properties */
+        justify-content: center;
+    }
+}
+
+/* Tablet Layout */
+@media only screen and (min-width: 600px) and (max-width: 992px) {
+    #bottomItemContainer {
+        /* Content Layout Properties */
+        flex-wrap: wrap;
+    }
+
+    #bottomPhotoContainer {
+        /* Self Layout Properties */
+        margin-bottom: 3.75rem;
+
+        /* Content Layout Properties */
+        flex-wrap: wrap;
+        row-gap: 1.25rem;
+    }
+
+    .bottomPhotoBox {
+        /* Self Layout Properties */
+        padding: 19rem 11.063rem;
+    }
+
+    #bottomPhotoBox1 {
+        /* UI Properties */
+        background: transparent url('@/icons/tablet/Clip_.png') 0% 0% no-repeat padding-box;
+    }
+
+    #bottomPhotoBox2 {
+        /* UI Properties */
+        background: transparent url('@/icons/tablet/Clip.png') 0% 0% no-repeat padding-box;
+    }
+
+    .bottomPhotoOverlay span {
+        /* UI Properties */
+        line-height: 0.6;
+    }
+
+    #bottomPhotoOverlay1 {
+        /* UI Properties */
+        background: transparent url('@/icons/tablet/Polygon_1.png') 0% 0% no-repeat padding-box;
+    }
+
+    #bottomPhotoOverlay2 {
+        /* UI Properties */
+        background: transparent url('@/icons/tablet/Layer_34.png') 0% 0% no-repeat padding-box;
+    }
+}
+
+/* Desktop Layout */
+@media only screen and (min-width: 992px) {
+    #bottomItemContainer {
+        /* Content Layout Properties */
+        display: flex;
+        flex-wrap: wrap;
+        row-gap: 4.4rem;
+    }
+
+    #bottomPhotoContainer {
+        /* Self Layout Properties */
+        width: 59.65%;
+    }
+
+    #bottomPhotoBox1 {
+        /* Self Layout Properties */
+        padding: 19rem 10.3rem;
+
+        /* UI Properties */
+        background: transparent url('@/icons/desktop/Clip_.png') 0% 0% no-repeat padding-box;
+    }
+
+    #bottomPhotoBox2 {
+        /* Self Layout Properties */
+        padding: 19rem 10.5rem;
+        
+        /* UI Properties */
+        background: transparent url('@/icons/desktop/Clip.png') 0% 0% no-repeat padding-box;
+    }
+
+    .bottomPhotoOverlay span {
+        /* Self Layout Properties */
+        top: 22.5rem;
+    }
+
+    #bottomPhotoOverlay1 {
+        /* UI Properties */
+        background: transparent url('@/icons/desktop/Polygon_1_bb.png') 0% 0% no-repeat padding-box;
+    }
+
+    #bottomPhotoOverlay2 {
+        /* UI Properties */
+        background: transparent url('@/icons/desktop/Layer_24.png') 0% 0% no-repeat padding-box;
```

Comment:
Coraz mniej zaczyna mnie przekonywać to jak rozbudowane te backgroundy są przy dodawaniu zdjęć. Spróbujmy chociaż w tym pliku zacząć od `background: url('...');` i w razie gdyby coś działało nie tak jak trzeba, to dodawać kolejne wartości, ale jestem pewny że przynajmniej o połowę można skrócić tą linijkę.

Reply:
Przyznam się bez bicia, że `background`y skopiowałem z Adobe Xd i od tego czasu o nich ani nie myślałem (chyba, że naprawdę musiałem coś zmienić). Biorę się zatem teraz za wielkie odchudzanie!